array.c

```diff
@@ -4901,15 +4963,33 @@ rb_ary_transpose(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.replace(other_ary)  -> ary
- *     ary.initialize_copy(other_ary)	-> ary
```

Comment:
I think it is OK you leave out #initialize_copy here as it is normally an "internal" method to implement #clone/#dup.